pkg/services/ngalert/api/api_convert_prometheus.go

```diff
@@ -28,9 +54,100 @@ func (srv *ConvertPrometheusSrv) RouteConvertPrometheusGetNamespace(c *contextmo
 }
 
 func (srv *ConvertPrometheusSrv) RouteConvertPrometheusGetRuleGroup(c *contextmodel.ReqContext, namespaceTitle string, group string) response.Response {
-	return response.Error(501, "Not implemented", nil)
+	return response.YAML(http.StatusOK, apimodels.PrometheusRuleGroup{})
```

Comment:
Just to make the `mimirtool rules load` work. It first checks if the group exists, and if the endpoint returns 501 it fails.

Reply:
Added, but this comment is quite temporary, after this PR I'll send another one for the reading APIs.